_data/import-public.rb

```diff
@@ -9,12 +9,60 @@
 require 'json'
 
 DATA_DIR = File.dirname __FILE__
+TEAM_DIR = "_team"
 TEAM_API_BASE_URL = 'https://team-api.18f.gov/public/api/'
+TEAM_ENDPOINT = 'team'
+MEMBER_PROPERTIES = [
```

Comment:
is that your preferred style? I tend to stay away from `%w` because I sometimes forget the difference between it and `%i` and all the other fancy shorthands in Ruby. This would be a good convo to have in https://github.com/18F/engineering-best-practices